templates/navigation/navigation.html

```diff
@@ -75,27 +75,25 @@
              aria-label="Toggle search"></a>
         </li>
       </ul>
-      {% if not page_bubble %}
-        <div class="p-navigation__search">
-          <form class="p-search-box">
-            <input type="search"
-                   class="p-search-box__input"
-                   name="q"
-                   placeholder="Search our sites"
-                   required=""
-                   aria-label="Search our sites" />
-            <button type="reset" class="p-search-box__reset">
-              <i class="p-icon--close"></i>
-            </button>
-            <button type="submit" class="p-search-box__button">
-              <i class="p-icon--search"></i>
-            </button>
-          </form>
-        </div>
-      {% endif %}
+      <div class="p-navigation__search">
+        <form class="p-search-box">
+          <input type="search"
+                  class="p-search-box__input"
+                  name="q"
```

Comment:
can we give it a more descriptive name?

Reply:
Naming a search input "q" is a widely adopted industry standard